### PR TITLE
Fix wrong style for bv-app-version info

### PR DESCRIPTION
**Pull request by danielgomezrico**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,9 +5,9 @@ Code styles, and tools for different languages
...
 - Clients should send the next headers on every request to our api:
 
-  * `bv-app-version`: `platform/typeApp/version/build` (***each value separated by a `/`***) _where_:
+  * `bv-app-version`: `platform/typeApp/version/build` (**each value separated by `/`**) where:
 
-    - `platform`***(mandatory)*** : platform who make the request in lower case (ej: ios, android)
-    - `typeApp`***(mandatory)*** : app profile who make the request in lower case (ej: customer, driver, admin) if there are only one app could be mobile for mobile apps
-    - `version`***(mandatory)*** : current app version following semver standard (x.y.z)
-    - `build`***(optional)*** : current build version, could be number or letters
+    - `platform`: platform who make the request in lower case (ej: ios, android)
```
Comment (edwaraco) [inline README.md:10]: I prefer to know what is mandatory.

**Review by edwaraco — CHANGES_REQUESTED** [review]
@caipivara I prefer to keep the current style.

File: README.md (hunk 1 above)
Comment (danielgomezrico) [reply README.md:10]: @earayo I prefer say just whats optional like swagger models :P, but the fix is not for that, is to fix a markdown issue

**Review by orendon — APPROVED** [review]
I like this 👍

**Review by Busta117 — APPROVED** [review]
👍

**Pull request merged**